examples/migration/CreateCompleteCampaignAdWordsApiOnly.py

```diff
@@ -0,0 +1,245 @@
+KEYWORDS_TO_ADD = ["mars cruise", "space hotel" ]
+
+
+def createCampaignBudget(client):
+    """Creates a new budget and returns the newly created budget id.
+
+    Args:
+        client: An instance of the Adwords client
+
+    Returns:
+        (str)Budget id of the newly created budget.
+    """
+    budget_service = client.GetService('BudgetService', version='v201809')
+    budget = {
+        'name' : 'Interplanetary Cruise Budget #{}'.format(uuid.uuid4()),
+        'amount': {
+            'microAmount' : '50000000'
+        },
+        'deliveryMethod' : 'STANDARD'
+    }
+    budget_operations = [{
+        'operator': 'ADD',
+        'operand': budget
+    }]
+    # Add budget.
+    results = budget_service.mutate(budget_operations)
+    createdBudget = results['value'][0]
+    print("Budget with ID {} and name {} was created".format(
+           createdBudget['budgetId'], createdBudget['name']))
+    return createdBudget['budgetId']
+
+
+def createCampaign(client, budgetId):
+    """Creates a new campaign and returns the newly created campaign id.
+
+    Args:
+        client: An instance of the Adwords client
+        budgetId: (str) Budget id to be referenced while creating Campaign
+
+    Returns:
+        (str)Campaign id of the newly created Campaign.
+    """
+    campaign_service = client.GetService('CampaignService', version='v201809')
+    campaign = {
+        'name': 'Interplanetary Cruise #{}'.format(uuid.uuid4()),
+        'advertisingChannelType': 'SEARCH',
+        # Recommendation: Set the campaign to PAUSED when creating it to stop the
+        # ads from immediately serving. Set to ENABLED once you've added
+        # targeting and the ads are ready to serve.
+        'status': 'PAUSED',
+        'biddingStrategyConfiguration': {
+            'biddingStrategyType': 'MANUAL_CPC',
+        },
+        'startDate': (datetime.datetime.now() +
+        datetime.timedelta(1)).strftime('%Y%m%d'),
+        'endDate': (datetime.datetime.now() +
+        datetime.timedelta(365)).strftime('%Y%m%d'),
+        # Budget (required) - note only the budget ID is required.
+        'budget': {
+            'budgetId': budgetId
+        },
+        'networkSetting': {
+            'targetGoogleSearch': 'true',
+            'targetSearchNetwork': 'true',
+        }
+    }
+    campaign_operations = [{
+        'operator': 'ADD',
+        'operand': campaign
+    }]
+    results = campaign_service.mutate(campaign_operations)
+    createdCampaign = results['value'][0]
+    print("CreatedCampign with ID {} and name {} was created".format(
+           createdCampaign['id'], createdCampaign['name']))
+    return createdCampaign['id']
+
+
+def createAdGroup(client, campaign_id):
+    """Creates a new adgroup and returns the newly created adgroup id.
+
+    Args:
+        client: An instance of the Adwords client
+        campaign_id: (str) Campaign id to be referenced while creating Adgroup
```

Comment:
Make sure to have consistent punctuation for all comments throughout, so have period at the end of each sentence.